### DISCUSSION TITLE
"error": "Failed to detect base model

### DISCUSSION THREAD

**Discussion by jayburton84** [Acly/krita-ai-diffusion#948] [Q&A]
Apologies, I'm new to this and I've been scouring the discussion board for an answer, but I've had no luck solving my issue. Unless I'm missing something, it would appear that I'm not able to utilize any model checkpoints added after initial installation.

I'm running a locally managed server. SD 1.5 and XL workloads are installed, along with the five recommended/default checkpoints (DreamShaper, JuggernautXL, etc) and all control extensions.

![image](https://github.com/user-attachments/assets/0b050eb3-41aa-4627-8d08-f0bc49f1de3a)

I have downloaded about a dozen new models from Civitai - all with a reported base model of SD 1.5 or SDXL - and I placed the .safetensors files here: \krita\ai_diffusion\server\ComfyUI\models\checkpoints. This is the same location that the five recommended/default checkpoints live. Those five default checkpoints are listed in the Model Checkpoint dropdown, but none of the new checkpoints that I downloaded are appearing. I've tried deleting and redownloading in case the files were incomplete.

![image](https://github.com/user-attachments/assets/ee54f8ef-b41d-4b9f-bbc4-f2f932736ec1)

When I open `127.0.0.1:8188/etn/model_info`, every new checkpoint that I added is listed with the following:
`"base_model": "unknown",
        "error": "Failed to detect base model: argument of type 'NoneType' is not iterable"`

Whereas the original recommended checkpoints have a base_model correctly identified as either sd15 or sdxl.

![image](https://github.com/user-attachments/assets/3a21fbed-9ad7-4ee4-9c48-fc1f67f3e870)

I've tried uninstalling and reinstalling, and I also made sure that comfyui-tooling-nodes was replaced with the most updated release. I've watched a bunch of tutorial videos, and everyone makes it look as easy as dragging those .safetensors into the `models/checkpoints` folder and then hitting refresh, but for some reason I'm not having the same experience. Any advice?

**Comment by Acly** [comment]
It looks like the models you downloaded are LoRA, not Checkpoints. They are small extensions to checkpoints instead of an entire model.
To use them put them in models/loras folder and add them in the Lora section of plugin's style settings.

**Reply by SargeRooster** [reply]
I am also running into the same issue. I have added models that are labels "checkpoint" into the checkpoint folder and it does not recognise them. I can restart the program and it doesn't pop up. Is there a setting that I am missing?
